BuildHelpers/Public/Get-BuildVariable.ps1

```diff
@@ -160,21 +160,21 @@ function Get-BuildVariable {
         'CI_COMMIT_SHA' {
             if($WeCanGit)
             {
-                Invoke-Git @IGParams -Arguments "log --format=%B -n 1 $( (Get-Item -Path "ENV:$_").Value )"
+                Invoke-Git @IGParams -Arguments "log --format=%B -n 1"
```

Comment:
So!  In this section (getting commit message), the deleted bit is the commit sha.  If you leave this off, AFAICT it will get the current commit.  Potential corner case:  Someone has been moving HEAD around in their build, but still want to refer to the commit message of the build's commit.

Do these lines ever error out for you?  Maybe I'm grabbing a wrong environment variable?  Intention in this section is to grab commit to do this: https://stackoverflow.com/a/3357357